SWIG/calendars.i

```diff
@@ -271,6 +272,12 @@ namespace QuantLib {
     };
 
     class TARGET : public Calendar {};
+
+    class Thailand : public Calendar {
+      public:
```

Comment:
The default constructor is implied, so it's not necessary to export it explicitly (see most of the other calendars).